e3db/types/__init__.py

```diff
@@ -12,3 +12,6 @@
 from .search import Search
 from .search import Params
 from .search import Range
+from .note import Note
+from .note_keys import NoteKeys
+from .note_options import NoteOptions
```

Comment:
Perhaps a newline would look good here at the end of the file?

Reply:
Added